SolarTherm/Models/Fluid/HeatExchangers/Boiler_Basic.mo

```diff
@@ -0,0 +1,62 @@
+within SolarTherm.Models.Fluid.HeatExchangers;
+
+model Boiler_Basic//Simplify Boiler
```

Comment:
please give a bit of context on intended use/general use of this model. is it a steam generator to be powered from molten salt, in the primary use case?